### DISCUSSION TITLE
Change Color of selected Date

### DISCUSSION THREAD

**Discussion by Kiriakos41** [ME-MarvinE/XCalendar#48] [Q&A] [answered]
Any Help with styles?
Can i change it to blue?
![ss](https://user-images.githubusercontent.com/70096169/172152919-9e40104b-a36d-41cc-96c2-7b838280d0a4.jpg)

**Comment by ME-MarvinE** [comment, marked as answer]
Yes. Using the [CalendarDayView](https://github.com/ME-MarvinE/XCalendar/wiki/CalendarDayView) control, you can change the BorderColor, BackgroundColor, TextColor, Command, and CommandParameter for each DayState.  
  
You would put the CalendarDayView in the [`DayTemplate`](https://github.com/ME-MarvinE/XCalendar/wiki/CalendarView#daytemplate) property of the CalendarView. There is an example in the sample app ([Forms example](https://github.com/ME-MarvinE/XCalendar/blob/master/XCalendarFormsSample/XCalendarFormsSample/Views/UsingCalendarDayViewExamplePage.xaml), [MAUI example](https://github.com/ME-MarvinE/XCalendar/blob/master/XCalendarMauiSample/Views/UsingCalendarDayViewExamplePage.xaml)).  
  

To give a summary of each DayState using your image, the first day you point to is `Today`, the second is `Selected`, all the others with black text color are `CurrentMonth`, and the grey ones are `OtherMonth`.

**Reply by Kiriakos41** [reply]
Thank you so much!
